modules/spring-config/src/main/java/org/mule/config/spring/parsers/AbstractHierarchicalDefinitionParser.java

```diff
@@ -47,6 +48,10 @@ protected String getParentBeanName(Element element)
         return ((Element) element.getParentNode()).getAttribute(ATTRIBUTE_NAME);
     }
 
+    protected Element getParentNode(Element element){
+        return ((Element) element.getParentNode());
```

Comment:
you should have better handling for the case in which the parent is not an element.